packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -79,17 +81,24 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 	hasPurchasedTheme = false,
 	onCheckout,
 	verticalId,
+	currentPlanFeatures,
 } ) => {
 	const { __ } = useI18n();
 	const { style_variations = [], is_premium: isPremium = false } = design;
 	const shouldUpgrade = isPremium && ! isPremiumThemeAvailable && ! hasPurchasedTheme;
+	const currentSiteCanInstallWoo = currentPlanFeatures?.includes( FEATURE_WOOP );
```

Comment:
Nit: As a personal preference, I strongly prefer that we explicitly define the value to match the expected type when we hit a null or undefined value. So in this case I'd prefer a boolean for all possible cases. This is definitely not blocking, but a personal preference.
```suggestion
	const currentSiteCanInstallWoo = currentPlanFeatures?.includes( FEATURE_WOOP ) ?? false;
```